internal/backend/db/dbgorm/triggers.go

```diff
@@ -14,24 +14,33 @@ import (
 
 func triggerToRecord(ctx context.Context, tx *tx, trigger sdktypes.Trigger) (*scheme.Trigger, error) {
 	connID := trigger.ConnectionID()
+	var conn sdktypes.Connection
+	var err error
+	var projID sdktypes.ProjectID
```

Comment:
nit
this is more "go" way
```suggestion
	var (
	       conn sdktypes.Connection
	       err error
	       projID sdktypes.ProjectID
	  )
```